frontend/src/components/pages/consumers/Modals.tsx

```diff
@@ -741,173 +777,144 @@ export class DeleteOffsetsModal extends Component<{
+                        Topics:
+                      </Text>{' '}
+                      {offsetsByTopic.length}
+                    </Text>
 
-    const toastMsg = 'Deleting offsets';
-    const toastRef = toast({
-      status: 'loading',
-      description: `${toastMsg}...`,
-      duration: null,
-    });
-    try {
-      if (this.props.mode === 'group') {
-        await api.deleteConsumerGroup(group.groupId);
-      } else {
-        const deleteRequest = createDeleteRequest(offsets);
-        const deleteResponse = await api.deleteConsumerGroupOffsets(group.groupId, deleteRequest);
-        const errors = deleteResponse
-          .map((t) => ({
-            ...t,
-            partitions: t.partitions.filter((x) => x.error),
-          }))
-          .filter((t) => t.partitions.length > 0);
-        if (errors.length > 0) {
-          console.error('backend returned errors for deleteOffsets', { request: deleteRequest, errors: errors });
-          throw { request: deleteRequest, errors: errors };
-        }
-      }
+                    <Text>Are you sure?</Text>
+                  </Flex>
+                </Box>
+              )}
 
-      toast.update(toastRef, {
-        status: 'success',
-        duration: 2000,
-        description: `${toastMsg} - done`,
-      });
-    } catch (err) {
-      console.error(err);
-      toast.update(toastRef, {
-        status: 'error',
-        duration: 2000,
-        description: `${toastMsg} - failed`,
-      });
-      showErrorModal(
-        'Delete offsets',
-        <span>
-          Could not delete selected offsets in consumer group <span className="codeBox">{group.groupId}</span>.
-        </span>,
-        toJson(err, 4),
-      );
-    } finally {
-      api.refreshConsumerGroups(true);
+              {mode === 'topic' && (
+                <Box>
+                  <Text>Group offsets will be deleted for topic:</Text>
+                  <List fontWeight="600" my={2}>
+                    <ListItem>
+                      Topic: <span className="codeBox">{offsetsByTopic[0].topicName}</span>
+                    </ListItem>
+                    <ListItem>
+                      {offsets.length} {singlePartition ? 'Partition' : 'Partitions'}
+                    </ListItem>
+                  </List>
+                </Box>
+              )}
 
-      const remainingOffsets = group.topicOffsets.sum((t) => t.partitionOffsets.length) - offsets.length;
-      if (remainingOffsets === 0) {
-        // Group is fully deleted, go back to list
-        appGlobal.history.replace('/groups');
-      } else {
-        this.props.onClose();
-      }
-    }
+              {mode === 'partition' && (
+                <Box>
+                  <Text>Group offsets will be deleted for partition:</Text>
+                  <UnorderedList fontWeight="600" my={2}>
+                    <ListItem>
+                      Topic: <span className="codeBox">{offsetsByTopic[0].topicName}</span>
```

Comment:
Why do we need to continue using classnames? Any way we can use `<Code>` component or similar?